config/database.go

```diff
@@ -21,6 +25,10 @@ func DbConfig() *mongo.Client {
 		log.Fatal(err)
 	}
 	log.Println("Connected to MongoDB!")
-	return client
+	mongoClient = client
+	return nil
```

Comment:
@GuildGamer does this not affect what has been done on the auth, did you check the impact?

Reply:
The client was not used in the auth middleware but used in the user service, I updated accordingly in the user service though so it works fine.